uint32 time_stamp

float32[3] pos_sp

float32[3] vel_sp

float32[3] vel_ff

float32[3] acc_ff

float32 pitch_sp
float32 roll_sp
float32 yaw_sp

float32 throttle
